src/util/system.h

```diff
@@ -453,6 +444,16 @@ class ArgsManager
     void LogArgs() const;
 
 private:
+
```

Comment:
849ea30efe96b29ec3023d10b077ffb4f7fa0f86: According to clang-format this newline should be removed

Reply:
Would it make sense to add a clang-format linter to Cirrus or maybe to `lint-all.sh` to enforce proper code style?

Reply:
There are cases where clang-format is "wrong", so we can't enforce this. Maybe suggestions could make sense, but they might be noisy.